subsys/net/l2/openthread/Kconfig

```diff
@@ -96,8 +96,13 @@ module-str = Log level for OpenThread driver
 module-help = Enables OpenThread driver to output debug messages.
 source "subsys/net/Kconfig.template.log_config.net"
 
+config OPENTHREAD_THREAD_PREEMPTIVE
+	bool "Set Openthread thread to be preemptive"
+	default n
```

Comment:
Actually, there's no need for `default n`, please remove.

Reply:
removed